.env.example

```diff
@@ -24,7 +24,7 @@ REACT_APP_USE_RECAPTCHA=
 REACT_APP_RECAPTCHA_SITE_KEY=
 
 # has to be inserted in the env file to use plugins and other websocket based features.
-REACT_APP_BACKEND_WEBSOCKET_URL=ws://localhost:4000/graphql
+REACT_APP_BACKEND_WEBSOCKET_URL=
```

Comment:
Why was this changed when it's a known working default value?

Reply:
Changed it back to the default value.